scripts/battlefields/Riverne_Site_A01/ouryu_cometh.lua

```diff
@@ -0,0 +1,45 @@
+-----------------------------------
+-- Area: Riverne Site #A01
+-- Name: Ouryu Cometh
+-- !pos 732.550 -32.5 -506.544 30
+-----------------------------------
+local riverneID = zones[xi.zone.RIVERNE_SITE_A01]
+-----------------------------------
+
+local content = Battlefield:new({
+    zoneId        = xi.zone.RIVERNE_SITE_A01,
+    battlefieldId = xi.battlefield.id.OURYU_COMETH,
+    maxPlayers    = 18,
+    levelCap      = 75,
+    timeLimit     = utils.minutes(30),
+    index         = 0,
+    area          = 1,
+    entryNpc      = 'Spatial_Displacement',
+    exitNpc       = { '_0u1', '_0u2', '_0u3' },
+    requiredItems =
+    {
+        xi.item.CLOUD_EVOKER,
+        wearMessage = riverneID.text.TIME_LIMIT_FOR_THIS_BATTLE_IS + 2,
+        wornMessage = riverneID.text.TIME_LIMIT_FOR_THIS_BATTLE_IS + 1,
+    },
+    experimental = true,
+})
+
+content:addEssentialMobs({ 'Ouryu' })
```

Comment:
addEssentialMobs should not be necessary since you're explicitly defining groups and death handlers